src/services/local/index.js

```diff
@@ -111,6 +112,11 @@ export default function(options){
     // Get our initialize service to that we can bind hooks
     const localService = app.service(options.localEndpoint);
 
+    // prevent regular service events from being dispatched
+    if (app.io || app.primus) {
```

Comment:
I'd probably go with `if(typeof localService.filter === 'function')`. We should probably also add warnings if someone tries to override those methods.